tests/sql/getSQLParams.test.js

```diff
@@ -2,7 +2,7 @@ const AWS = require('aws-sdk');
 
 const config = require('../../src/config');
 
-const getConnectionParams = require('../../src/SQL/getConnectionParams');
+const getConnectionParams = require('../../src/sql/getConnectionParams');
```

Comment:
This one has a couple of names that I didn't change, could you change them too? Rename all the `getSQLParams` and `createSQLClient` to `getConnectionParams`